src/Integrations/Integrations/PDO/PDOIntegration.php

```diff
@@ -149,6 +164,7 @@ private static function parseDsn($dsn)
     {
         $engine = substr($dsn, 0, strpos($dsn, ':'));
         $tags = ['db.engine' => $engine];
+        $tags[Tag::DB_SYSTEM] = self::DB_DRIVER_TO_SYSTEM[$tags['db.engine']];
```

Comment:
I'd probably do something more like this:
```suggestion
        $tags[Tag::DB_SYSTEM] = isset(self::DB_DRIVER_TO_SYSTEM[$engine]) ? self::DB_DRIVER_TO_SYSTEM[$engine] : 'other_sql';
```

This is a little bit more resilient, and it also uses `$engine` directly instead of using `$tags['db.engine']` (which gets its info from `$engine`).

Reply:
Good idea! Just added this suggestion